google/cloud/storage/tests/testbench.py

```diff
@@ -416,6 +420,9 @@ def get_revision(self, request):
                 'Precondition Failed: generation %s not found' % generation)
         return version
 
+    def get_revision_by_generation(self, generation):
```

Comment:
Please add a short docstring, and do you mind moving close to `get_latest()`?  The two functions seem related makes sense to keep them together (though I do not claim that this file is brilliantly organized).